plugins/modules/cloud/misc/terraform.py

```diff
@@ -32,6 +32,11 @@
       - The path to the root of the Terraform directory with the
         vars.tf/main.tf/etc to use.
     required: true
+  log_error_path:
+    description:
+      - The path to the root of the Terraform directory with the
+        vars.tf/main.tf/etc to use.
+    required: false
```

Comment:
```suggestion
    type: path
    version_added: 0.2.0
```
Type should always be in the docs (it's probably not because the module hasn't been updated since that rule was introduced, and the sanity checks don't fail because the rule is ignored for this file - for new options this should really be added though).

The `version_added` is since the next release of the collection will be 0.2.0. If the PR isn't merged until that release happens, the next release will be 1.0.0.

Reply:
noted